test/mpi/pt2pt/testlist.in

```diff
@@ -1,3 +1,4 @@
+sendself 1 test/mpi/pt2pt/sendself arg=-type=MPI_C_DOUBLE_COMPLEX arg=-sendcnt=8192 arg=-recvcnt=8192 arg=-seed=47 arg=-testsize=1
```

Comment:
Oh, we probably want to increase the testsize to e.g. 100 to cover more tests. Also  I think we should add a second test with `np = 2`. A single process communication goes to the self communication path and it will skip the netmod.

Reply:
Just pushed those changes.